Source/API/CBLManager.m

```diff
@@ -446,6 +446,7 @@ - (BOOL) replaceDatabaseNamed: (NSString*)databaseName
                                                toPath: dstAttachmentsPath
                                                 error: outError]) &&
             [db open: outError] &&
+            [db createLocalCheckpointDocument] &&
```

Comment:
-createLocalCheckpointDocument should take an out NSError parameter. Otherwise, if it fails (returns NO) then this method will return NO but won't have set its own outError to any valid value.